README.md

```diff
@@ -344,6 +345,14 @@ To add a resource or contribute, please check out our contribution guide [here](
 
 ---
 
+## Ionic
+
+- [Getting Started with Ionic Framework](https://ionicframework.com/docs)
+- [Deploying Ionic Apps](https://ionicframework.com/docs/angular/your-first-app/6-deploying-mobile)
+- [Ionic Tips and Trick from Josh Morony Blog](https://www.joshmorony.com/tags/ionic/)
```

Comment:
A link to a blog is not allowed unless it's a continuous series of posts, contains extensive information, or the information in the blog post is truly different than others around the web. 

If this blog has posts that falls under these rules, then you can add them here. If not, then please remove this link.

Reply:
hi, i'd already change it, please do check the newest commit